.github/workflows/test.yaml

```diff
@@ -45,6 +45,8 @@ jobs:
       
       - name: Run tests
         run: |
+          mkdir -p /home/runner/work/dashboard/dashboard/.nyc_output
```

Comment:
The e2e tests were failing due to coverage failing.
Coverage was failing complaining that out.json didn't exist so I created it. The tests started passing but I'm not certain the coverage was actually getting generated.

Reply:
This should be automatically generated and if missing there must be some configuration issues or mismatching versions.
On a first glance, this is related to a lack of instrumentalization of the tests required to generate the coverage afterwards.
This has been added as [babel handler within Nuxt plugins as `'babel-plugin-istanbul'`](https://github.com/rancher/dashboard/blob/master/shell/nuxt.config.js#L545).

Reply:
Please do not forget to remove the change to the test, as it already should highlight an issue in a configuration of the project, rather than an issue in the configuration of the CI.